packages/cli/src/evaluation/test-runner/test-runner.service.ee.ts

```diff
@@ -57,20 +96,13 @@ export class TestRunnerService {
 		// Create pin data from the past execution data
 		const pinData = createPinData(workflow, pastExecutionData);
 
-		// Determine the start node of the past execution
-		const pastExecutionStartNode = getPastExecutionTriggerNode(pastExecutionData);
-
 		// Prepare the data to run the workflow
 		const data: IWorkflowExecutionDataProcess = {
-			destinationNode: pastExecutionData.startData?.destinationNode,
-			startNodes: pastExecutionStartNode
-				? [{ name: pastExecutionStartNode, sourceData: null }]
-				: undefined,
+			...this.getStartNodesData(workflow, pastExecutionData),
```

Comment:
We now omit `destioationNode` entirely. Is that intentional?

Reply:
Yes, we decided to run the entire wf, even if the past execution was partial (had destination node).